arangod/Aql/ExecutionEngine.cpp

```diff
@@ -175,7 +175,8 @@ Result ExecutionEngine::createBlocks(std::vector<ExecutionNode*> const& nodes,
     root(eb);
 
     // put it into our cache:
-    cache.emplace(en, eb);
+    auto [itr, emplaced] = cache.try_emplace(en, eb);
+    TRI_ASSERT(emplaced);
```

Comment:
behavior change... Is it _required_ that the emplace works here?

Reply:
Since blocks are pointers it is probably not required. But it would be nice if we would try just a single insert. check removed.